metrics/docker-compose.yml

```diff
@@ -1,18 +1,48 @@
 services:
   prometheus:
-    network_mode: "host" # -- only linux
     image: prom/prometheus
     container_name: prometheus
-    command:
-      - "--config.file=/etc/prometheus/prometheus.yml"
+    restart: unless-stopped
     ports:
       - 9090:9090
-    restart: unless-stopped
     volumes:
       - ./prometheus:/etc/prometheus
       - prom_data:/prometheus
+    command:
+      - "--config.file=/etc/prometheus/prometheus.yml"
+      # "--log.level=debug" # Not receiving metrics? Consider uncommenting
+
+    # Prometheus needs to be able to connect to the host network as "sig" isn't
+    # containerised. No other service should talk to the host.
+    extra_hosts:
+      - "host.docker.internal:host-gateway"
+
+  loki:
+    image: grafana/loki:3.0.0
+    container_name: loki
+    ports:
+      - "3100:3100"
```

Comment:
Is there a reason to expose these ports to the host system? This shouldn't be necessary just to connect from other containers in the same docker network.

Reply:
sobe reconfigured it to not use host mode anymore

Reply:
Yeah I noticed that. By default all the containers from a compose file share the same docker network with all ports exposed to the other containers in that network. This line exposes it to the host system, which seems unnecessary, unless you're connecting to it from outside of docker.